core/src/main/scala/pureconfig/error/ConfigReaderFailures.scala

```diff
@@ -21,4 +21,9 @@ object ConfigReaderFailures {
 
   def apply(configReaderFailure: ConfigReaderFailure): ConfigReaderFailures =
     new ConfigReaderFailures(configReaderFailure, List.empty[ConfigReaderFailure])
+
+  def apply(configReaderFailures: List[ConfigReaderFailure]): ConfigReaderFailures = {
+    require(configReaderFailures.length > 0)
```

Comment:
I don't like this, `ConfigReaderFailures` should be a `NonEmptyList` and should always be built from either just the head or the head and a tail. Why do we want to add this method that can throw exception?

Reply:
I've removed it in ba8e6e1.